spanner/test/cloudexecutor/executor/internal/inputstream/handler.go

```diff
@@ -110,11 +140,20 @@ func (h *CloudStreamHandler) startHandlingRequest(ctx context.Context, req *exec
 	}
 
 	// Get a new action handler based on the input action.
-	actionHandler, err := h.newActionHandler(inputAction, outcomeSender)
+	actionType, actionHandler, err := h.newActionHandler(inputAction, outcomeSender)
```

Comment:
`actionType` can be derived directly from `inputAction`. Passing it from newActionHandler is redundant and is hard to maintain.

My suggestion would be,
Create a new function and use it where required.
```
func getActionType(inputAction *executorpb.SpannerAction) string {
    // GET THE ACTION TYPE
    action := fmt.Sprintf("%T", inputAction.GetAction())
   // Here action will have output as `*executorpb.SpannerAction_Query`. Apply string processing to convert into required format
}

```

Reply:
Done.